components/automate-cs-bookshelf/habitat/config/sys.config

```diff
@@ -60,9 +60,9 @@
  {chef_secrets, [
                  {provider, chef_secrets_fd}
                 ]},
- {bookshelf, [{ip, "127.0.0.1"},
+ {bookshelf, [{ip, "{{cfg.bookshelf.virtual_ip}}"},
               {port, {{cfg.network.port}}},
-              {storage_type, sql},
+              {storage_type,{{cfg.bookshelf.storage_type}}},
```

Comment:
as this is a string, does it requires double quotes, can we check it once?

Reply:
Hey @YashviJain01 I suppose this is an atom and hence it will not require double quotes.